x-pack/plugins/observability_solution/apm/public/components/app/service_inventory/multi_signal_inventory/table/get_service_columns.tsx

```diff
@@ -254,7 +258,11 @@ export function getServiceColumns({
       sortable: true,
       dataType: 'number',
       align: RIGHT_ALIGNMENT,
-      render: (_, { metrics, serviceName }) => {
+      render: (_, { metrics, serviceName, signalTypes }) => {
+        if (isLogsSignal(signalTypes) && metrics.logErrorRate === undefined) {
```

Comment:
it can happen that the service doesn't have any errors. Should we also check  `metrics.logRate === undefined` if not only the logRate to display `NotAvailableLogsMetrics`, wdyt?